src/pages/transaction/transaction-show/transaction-show.ts

```diff
@@ -19,10 +19,10 @@ export class TransactionShowPage {
 
   public transaction: Transaction;
   public currentNetwork: Network;
-  public equivalentAmount: number = 0;
+  public equivalentAmount = 0;
```

Comment:
Why did you remove the type from some properties, but keep others? 🤔

Reply:
I think the rule is, if the type is clear when declaring the variable, i.e. has an immediate value (like `0`) the type has to be omitted, else it must be declared.